src/core_write.cpp

```diff
@@ -166,6 +167,16 @@ void TxToUniv(const CTransaction& tx, const uint256& hashBlock, UniValue& entry)
             o.pushKV("asm", ScriptToAsmStr(txin.scriptSig, true));
             o.pushKV("hex", HexStr(txin.scriptSig.begin(), txin.scriptSig.end()));
             in.pushKV("scriptSig", o);
+            if (!tx.wit.IsNull()) {
+                if (!tx.wit.vtxinwit[i].IsNull()) {
+                    UniValue txinwitness(UniValue::VARR);
+                    for (unsigned int j = 0; j < tx.wit.vtxinwit[i].scriptWitness.stack.size(); j++) {
+                        std::vector<unsigned char> item = tx.wit.vtxinwit[i].scriptWitness.stack[j];
+                        txinwitness.push_back(HexStr(item.begin(), item.end()));
+                    }
+                in.push_back(Pair("txinwitness", txinwitness));
```

Comment:
Fixed (and changed `push_back(Pair())` to `pushKV` to match other fields).

All test_bitcoin tests pass.